core/controllers/story_viewer.py

```diff
@@ -78,23 +77,3 @@ def get(self, story_id):
             'story_nodes': ordered_node_dicts
         })
         self.render_json(self.values)
-
-
-class StoryNodeCompletionHandler(base.BaseHandler):
```

Comment:
Why is this deleted?

Reply:
Since, the completion is now done as part of the PUT request in RecommendationsHandler.